closed/src/java.base/share/classes/sun/security/provider/NativeDigest.java

```diff
@@ -56,9 +57,24 @@
     private long bytesProcessed;
 
     private static NativeCrypto nativeCrypto;
+    private static Cleaner digestCleaner;
 
     static {
         nativeCrypto = NativeCrypto.getNativeCrypto();
+	digestCleaner = Cleaner.create();
```

Comment:
Do we *need* a distinct `Cleaner` object? Can we not reuse `CleanerFactory.cleaner()`?

Reply:
updated